src/components/tables/RowActions/Shared/Progress.tsx

```diff
@@ -41,6 +42,8 @@ function SharedProgress({
 }: SharedProgressProps) {
     const theme = useTheme();
 
+    const showErrors = state === ProgressStates.FAILED && error !== null;
```

Comment:
This was initially shared but then wasn't. Left it in as I can still see a future where progress does special things when showing errors.